src/internal/clusterstate/v2.8.0/clusterstate.go

```diff
@@ -10,6 +10,7 @@ func Migrate(state migrations.State) migrations.State {
 		Apply("Migrate collections.repos to pfs.repos", migrateRepos, migrations.Squash).
 		Apply("Migrate collections.branches to pfs.branches", migrateBranches, migrations.Squash).
 		Apply("Alter the pfs.commits schema", migrateCommitSchema, migrations.Squash).
+		Apply("Migrate collections.commits to pfs.commits", migrateCommitsFromCollections, migrations.Squash).
```

Comment:
What do you think about merging this with `"Alter the pfs.commits schema"` into a single migration? We can still call it `"Migrate collections.commits to pfs.commits"`, making the names consistent with the above.

Reply:
Can do